src/styles/blocks/product-catalog.scss

```diff
@@ -0,0 +1,29 @@
+.product-catalog {
+  padding-block: 50px;
+  padding-inline: 40px;
```

Comment:
The `padding-inline` property is not widely supported yet. It's better to use `padding-left` and `padding-right` instead.